kitchenci/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  puppet_version: "3.8.1"
+  mainfests_path: manifests
+  modules_path: modules
+
+platforms:
+  {platform-placeholder}
```

Comment:
Ok. But, I still can't see how `{platform-placeholder}` would be converted to `-name: centos-6.6`. Can you point that out to me?